playground/android/app/build.gradle

```diff
@@ -1,19 +1,12 @@
 apply plugin: "com.android.application"
+apply plugin: "com.facebook.react"
 
-project.ext.react = [
-        root     : "../../../",
-        entryFile: "index.js",
-        cliPath : "node_modules/react-native/cli.js",
-        bundleAssetName: "index.android.bundle",
-        bundleInAlpha: true,
-        bundleInBeta: true,
-        enableHermes: true,
-        hermesFlagsDebug:['-Xes6-proxy','-output-source-map'],
-        hermesFlagsRelease:['-output-source-map'],
-        hermesCommand: "../../../node_modules/react-native/sdks/hermesc/%OS-BIN%/hermesc",
-]
-
-apply from: "../../../node_modules/react-native/react.gradle"
+react {
+     root = file("../../../")
+}
+
+
+def jscFlavor = 'org.webkit:android-jsc:+'
 
 android {
     compileSdkVersion rootProject.ext.get("compileSdkVersion")
```

Comment:
consider adding a namespace here, like so:

`namespace "com.reactnativenavigation.playgroun` and remove `package="com.reactnativenavigation.playground"` from `AndroidManifest.xml` for the package name.